pkg/api/resource/suffix.go

```diff
@@ -83,6 +83,8 @@ func newSuffixer() suffixer {
 	// a suffix for 2^0.
 	sh.decSuffixes.addSuffix("", bePair{2, 0})
 
+	sh.decSuffixes.addSuffix("n", bePair{10, -9})
+	sh.decSuffixes.addSuffix("μ", bePair{10, -6})
```

Comment:
Question to reviewers: Should we just use `u` here, or is extended ASCII / unicode safe?